KestrelHttpServer.sln

```diff
@@ -61,6 +61,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TestResources", "TestResour
 		test\shared\TestResources\testCert.pfx = test\shared\TestResources\testCert.pfx
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Server.Kestrel.Libuv", "src\Microsoft.AspNetCore.Server.Kestrel.Libuv\Microsoft.AspNetCore.Server.Kestrel.Libuv.csproj", "{A76B8C8C-0DC5-4DD3-9B1F-02E51A0915F4}"
```

Comment:
As long as we're moving all of the code into different folders, would this be a good time to make project folder names shorter? Now that we're off PJ we can shorten folder names. Improves browsing code on GitHub and easier to use on commandline.